com.unity.ml-agents/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 ### Major Changes
 #### com.unity.ml-agents (C#)
 #### ml-agents / ml-agents-envs / gym-unity (Python)
+The minimum supported python version for ml-agents-envs was changed to 3.6.1. (#4244)
```

Comment:
3.6.1 was release on March 21, 2017; I think it's safe enough to require it now.
(Note that we already had this requirement for mlagents)